pytorch_lightning/strategies/strategies_registry.py

```diff
@@ -21,46 +21,46 @@
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
 
 
-class _TrainingTypePluginsRegistry(dict):
-    """This class is a Registry that stores information about the Training Type Plugins.
+class _StrategiesRegistry(dict):
+    """This class is a Registry that stores information about the Training Strategies.
 
-    The Plugins are mapped to strings. These strings are names that idenitify
-    a plugin, e.g., "deepspeed". It also returns Optional description and
-    parameters to initialize the Plugin, which were defined durng the
+    The Strategies are mapped to strings. These strings are names that identify
+    a strategy, e.g., "deepspeed". It also returns Optional description and
+    parameters to initialize the Strategy, which were defined durng the
     registration.
 
-    The motivation for having a TrainingTypePluginRegistry is to make it convenient
-    for the Users to try different Plugins by passing just strings
-    to the plugins flag to the Trainer.
+    The motivation for having a StrategiesRegistry is to make it convenient
+    for the Users to try different Strategies by passing just strings
+    to the strategy flag to the Trainer.
 
     Example::
 
-        @TrainingTypePluginsRegistry.register("lightning", description="Super fast", a=1, b=True)
+        @StrategiesRegistry.register("lightning", description="Super fast", a=1, b=True)
         class LightningPlugin:
```

Comment:
Could you update this line too :) <3 
```suggestion
        class LightningStrategy:
```

Reply:
Updated!